tools/ci/changed.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+patterns=()
+any_of=0
+
+while [[ $# -gt 0 ]]; do
+  case "$1" in
+    --any-of)
+      any_of=1
+      ;;
+    *)
+      patterns+=($1)
+      ;;
+  esac
+  shift
+done
+
+[[ ${#patterns[@]} -eq 0 ]] && echo 'No file pattern is specified, exiting' && exit 1
+
+changed_files=$(git diff --name-only origin/${ghprbTargetBranch:-master}..${ghprbActualCommit:-HEAD})
```

Comment:
Is this set `master` as the default target? If `ghprbTargetBranch` is missing?

Reply:
`ghprbTargetBranch` is always present in Jenkins environment, it should be the former branch here 
![image](https://user-images.githubusercontent.com/15965696/67443279-b95c1f80-f636-11e9-9b3f-b8eafea458fd.png)

Reply:
default to `master` is only for testing locally, it can be safely removed in Jenkins environment